splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/pages/Input/InputPage.jsx

```diff
@@ -48,49 +41,49 @@ function InputPage() {
 
     const generateModalDialog = () => {
         if (open) {
+            
             return (
                 < EntityModal
                     isInput
                     open={open}
                     handleRequestClose={handleRequestClose}
-                    handleSavedata={null}
                     serviceName={serviceName}
                     mode={MODE_CREATE}
                     formLabel={serviceLabel}
                 />
             );
         }
-            return null;
+        return null;
     }
 
     return (
         <>
             <ColumnLayout gutter={8}>
                 <ColumnLayout.Row style={{ padding: '5px 0px' }}>
                     <ColumnLayout.Column span={9}>
-                        <TitleComponent>{title}</TitleComponent>
-                        <SubTitleComponent>{description}</SubTitleComponent>
+                        <TitleComponent>{_(title)}</TitleComponent>
+                        <SubTitleComponent>{_(description)}</SubTitleComponent>
                     </ColumnLayout.Column>
                     {services && services.length > 1 &&
                         (<ColumnLayout.Column span={3} style={{ 'textAlign': 'right' }}>
                             <Dropdown toggle={toggle}>
-                                <Menu onClick={ (event) => {
-                                        const findname =  services[services.findIndex(x => x.title ===event.target.innerText)].name;
-                                        setserviceLabel(`Add ${event.target.innerText}`)
-                                        setserviceName(findname);
-                                        handleRequestOpen();
-                                        }
+                                <Menu onClick={(event) => {
+                                    const findname = services[services.findIndex(x => x.title === event.target.innerText)].name;
+                                    setserviceLabel(`Add ${event.target.innerText}`)
```

Comment:
a variable should be in camelcase: setserviceLabel

Reply:
done